packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -6,17 +6,26 @@
  */
 
 import { inject, injectable } from "inversify";
-import { QuickOpenModel, QuickOpenItem, QuickOpenMode, QuickOpenService, OpenerService } from '@theia/core/lib/browser';
+import { QuickOpenModel, QuickOpenItem, QuickOpenMode, QuickOpenService, OpenerService, KeybindingRegistry, Keybinding } from '@theia/core/lib/browser';
 import { FileSystem, FileStat } from '@theia/filesystem/lib/common/filesystem';
 import { WorkspaceService } from '@theia/workspace/lib/browser/workspace-service';
 import URI from '@theia/core/lib/common/uri';
 import { FileSearchService } from '../common/file-search-service';
 import { CancellationTokenSource } from '@theia/core/lib/common';
 import { LabelProvider } from "@theia/core/lib/browser/label-provider";
+import { Command } from '@theia/core/lib/common';
+
+export const quickFileOpen: Command = {
```

Comment:
Is there a reason for moving this here?

Reply:
If not moving, it createsa circular dependency